src/FacebookLoginTrait.php

```diff
@@ -43,10 +43,6 @@ public function loginFacebook(Request $request)
                  * Check if the user has already signed up.
                  */
                 $userModel = config('auth.providers.users.model');
-
-                /**
-                 * Create a new user if they haven't already signed up.
-                 */
                 $facebook_id_column = config('facebook.registration.facebook_id', 'facebook_id');
                 $name_column        = config('facebook.registration.name', 'name');
```

Comment:
If we are not creating a user we don't need to assign these variables, maybe move them down? (We only need to know the facebook id column.